src/networkcloud/azext_networkcloud/operations/custom_properties.py

```diff
@@ -0,0 +1,198 @@
+            _schema.operations = cls._schema_operation_status_result_read.operations
+            _schema.percent_complete = cls._schema_operation_status_result_read.percent_complete
+            _schema.resource_id = cls._schema_operation_status_result_read.resource_id
+            _schema.start_time = cls._schema_operation_status_result_read.start_time
+            _schema.status = cls._schema_operation_status_result_read.status
+            return
+
+        cls._schema_operation_status_result_read = _schema_operation_status_result_read \
+            = AAZObjectType()
+
+        operation_status_result_read = _schema_operation_status_result_read
+        operation_status_result_read.end_time = AAZStrType(
+            serialized_name="endTime",
+        )
+        operation_status_result_read.error = AAZObjectType()
+        cls._build_schema_error_detail_read(operation_status_result_read.error)
+
+        operation_status_result_read.properties = AAZObjectType()
+        cls._build_schema_properties_read(
+            operation_status_result_read.properties)
+
+        operation_status_result_read.id = AAZStrType()
+        operation_status_result_read.name = AAZStrType()
+        operation_status_result_read.operations = AAZListType()
+        operation_status_result_read.percent_complete = AAZFloatType(
+            serialized_name="percentComplete",
+        )
+        operation_status_result_read.resource_id = AAZStrType(
+            serialized_name="resourceId",
+            flags={"read_only": True},
+        )
+        operation_status_result_read.start_time = AAZStrType(
+            serialized_name="startTime",
+        )
+        operation_status_result_read.status = AAZStrType(
+            flags={"required": True},
+        )
+
+        operations = _schema_operation_status_result_read.operations
+        operations.Element = AAZObjectType()
+        cls._build_schema_operation_status_result_read(operations.Element)
+
+        _schema.end_time = cls._schema_operation_status_result_read.end_time
+        _schema.error = cls._schema_operation_status_result_read.error
+        _schema.properties = cls._schema_operation_status_result_read.properties
+        _schema.id = cls._schema_operation_status_result_read.id
+        _schema.name = cls._schema_operation_status_result_read.name
+        _schema.operations = cls._schema_operation_status_result_read.operations
+        _schema.percent_complete = cls._schema_operation_status_result_read.percent_complete
+        _schema.resource_id = cls._schema_operation_status_result_read.resource_id
+        _schema.start_time = cls._schema_operation_status_result_read.start_time
+        _schema.status = cls._schema_operation_status_result_read.status
+
+    # Custom handling of response will print out the output head and the result URL
+    # it will also save files into output directory if provided
+    def _output(self, *args, **kwargs):
```

Comment:
It seems the _output should be @staticmethod
```suggestion
    @staticmethod
    def _output(self, *args, **kwargs):
```

Reply:
Does the tests can cover this function? It seems not work right now.